app/uk/gov/hmrc/cataloguefrontend/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -58,6 +58,36 @@ object RepoType {
     }
 }
 
+sealed trait ServiceType {def toString: String}
```

Comment:
Can the model be moved to `model` since it's part of catalogue's domain, not just a local domain to `TeamsAndRepositoriesConnector`

Reply:
I see this is where `RepoType` is defined. We can leave it here for now, and clean up the model later